src/test/user.test.js

```diff
@@ -89,3 +90,32 @@ describe('GET /users/requests', () => {
     expect(response).to.have.status(404);
   });
 });
+
+describe('POST /users/request/stats', () => {
+  it('should successfully return the number of trip request count', async () => {
+    const queryObj = {
+      startDate: formatDate(Date.now()),
```

Comment:
Please explain what's happening here, isn't startdate === endDate here?

Reply:
You're right @Baystef, I forgot to update the test after I made some changes in the validation